diff --git a/node_modules/builder-util/out/util.js b/node_modules/builder-util/out/util.js
--- a/node_modules/builder-util/out/util.js
+++ b/node_modules/builder-util/out/util.js
@@ -253,6 +253,9 @@ function doSpawn(command, args, options, extraOptions) {
     }
     logSpawn(command, args, options);
     try {
+        if (process.platform === "win32" && (command.endsWith(".exe") || command.endsWith(".EXE"))) {
+            return (0, child_process_1.spawn)(command, args, options);
+        }
         return (0, cross_spawn_1.spawn)(command, args, options);
     }
     catch (e) {
